model.py

```diff
@@ -2738,27 +2743,29 @@ class Edition(Base):
         PERIODICAL_MEDIUM : u"http://schema.org/PublicationIssue",
         MUSIC_MEDIUM :  u"http://schema.org/MusicRecording",
         VIDEO_MEDIUM :  u"http://schema.org/VideoObject",
+        IMAGE_MEDIUM :  u"http://schema.org/ImageObject",
+        COURSEWARE_MEDIUM : u""
```

Comment:
Let's put `http://schema.org/Course` for this.

Reply:
Ok, thank you !